optuna/storages/rdb/models.py

```diff
@@ -161,6 +162,26 @@ def find_by_id(cls, trial_id, session):
 
         return trial
 
+    @classmethod
+    def find_max_value_trial(cls, study_id, session):
+        # type: (int, orm.Session) -> TrialModel
```

Comment:
Could you apply code style fixes according to https://github.com/optuna/optuna/wiki/Coding-Style-Conventions#empty-lines-after-type-hints ?

Reply:
done!